### ISSUE TITLE
Documentation: Include docs for switching IDA 7.4's IDAPython from Python3 to Python2

### ISSUE THREAD

**Issue by arizvisa** [arizvisa/ida-minsc#41]
IDA 7.4 includes the option to enable the Python3 or Python2 IDAPython plugin by default. It would be great to update the docs to reference how to enable Python2 for IDAPython in a user's installation.

A friend pasted me the following filenames that contain the plugin libraries to rename for his platform.
```
 1994  mv idapython.dylib idapython.3.disabled
 1996  mv idapython64.dylib idapython64.3.disabled
 1997  mv idapython.2.disabled idapython.dylib
 1998  mv idapython64.2.disabled idapython64.dylib
/Applications/IDA Pro 7.4/ida.app/Contents/MacOS/plugins
```

**Comment by arizvisa** [comment]
This issue is fixed by PR #87. The changes will make their way into the online documentation during the next release (which will probably happen in an hour or so after I rebuild the docs).

**Comment by arizvisa** [comment]
Closing this issue due to release being completed (new docs are at https://arizvisa.github.io/ida-minsc/).
